Examples/Physics_applications/QED/inputs.2d

```diff
@@ -78,6 +86,10 @@ photons.uz_m = 500.
 photons.ux_th = 2.
 photons.uy_th = 2.
 photons.uz_th = 50.
+##########QED####################
```

Comment:
Here do_qed_breit_wheeler enables the Breit Wheeler process, which concerns only photons. This process is implemented only for PhotonParticleContainers, so there is no risk of doing something unphysical. However it could be beneficial for the user to receive an error message if he/she tries to enable this process for electrons...

Reply:
Where would you suggest to add that? PhotonParticlesContainers are also PhysicalParticleContainers and they call the constructor of the base class... I could easily add a test in PhotonParticlesContainer to check for the do_qed_quantum_sync flag however

Reply:
Where would you add that? PhotonParticleContainers are also PhysicalParticleContainers and they call the constructor of the base class. The easy thing (just pushed) is to add a check for  do_qed_quantum_sync in the constructor of PhotonParticleContainer

Reply:
Yeah, I see the issue. If you want to throw an error when a `PhysicalParticleContainer` that is not a `PhotonParticleContainer` has `do_qed_breit_wheeler=1`, we could add a new attribute in the `PhysicalParticleContainer`, like `is_photon`, and set it to the appropriate value. Then we could add this piece of code into the `PhysicalParticleContainer` constructor
```c++
bool do_qed_breit_wheeler;
pp.query("do_qed_breit_wheeler", do_qed_breit_wheeler);
if (is_photon == 0){
    AMREX_ALWAYS_ASSERT_WITH_MESSAGE(
        do_qed_breit_wheeler == 0,
        "ERROR: do_qed_breit_wheeler can be 1 for species listed in particles.photon_species only!");
}
```